js/feature/mergedTrack.js

```diff
@@ -77,6 +79,8 @@ MergedTrack.prototype.draw = function (options) {
 
     dataRange = autoscale(options.genomicState.chromosome.name, mergedFeatures);
 
+    //IGVGraphics.fillRect(options.context, 0, options.pixelTop, options.pixelWidth, options.pixelHeight, {'fillStyle': "rgb(255, 255, 255)"});
+
```

Comment:
I saw a lot of the tracks do ```IGVGraphics.fillRect(options.context, 0, options.pixelTop, options.pixelWidth, options.pixelHeight, {'fillStyle': "rgb(255, 255, 255)"});```
before rendering. I added it here, but then commented it out because everything seemed to work without this. Should I reenable it?

Reply:
Not sure myself,  I'm just now pivoting back to igv.js (from IGV) and will give this a good look today and tomorrow.